Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -360,13 +360,13 @@ theorem constantCoeff_smul {S : Type*} [Semiring S] [Module R S] (φ : PowerSeri
 
 theorem smul_eq_C_mul (f : R⟦X⟧) (a : R) : a • f = C R a * f := by
   ext
-  simp
+  rw [LinearMapClass.map_smul, smul_eq_mul, coeff_C_mul]
 set_option linter.uppercaseLean3 false in
 #align power_series.smul_eq_C_mul PowerSeries.smul_eq_C_mul
 
 @[simp]
 theorem coeff_succ_mul_X (n : ℕ) (φ : R⟦X⟧) : coeff R (n + 1) (φ * X) = coeff R n φ := by
-  simp only [coeff, Finsupp.single_add]
+  rw [coeff, Finsupp.single_add]
```

Comment:
This change is equally readable; I would say it's fine. (If it's not faster, it might not be worth spending time *making* this change in other files, but I don't see this as negative.)